beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/hive/HiveClient.java

```diff
@@ -105,6 +106,18 @@ public boolean tableExists(String databaseName, String tableName) {
     }
   }
 
+  @Override
+  public boolean tableHasProperty(String databaseName, String tableName, String property, String value) {
+    try {
+      Table table = client.getTable(databaseName, tableName);
+      return table.getParameters().get(property).equals(value);
```

Comment:
this needs a bit safer, getParameters can be null, and get(property) can also return null I think.

Reply:
yeah, I'll add some checks, but I think the params will always exist in hive tables cause the size, num of rows etc are generated there; the `get(property)` can return null, which won't be equal to the passed value, so it will return false

Reply:
updated